src/component/Context.jsx

```diff
@@ -0,0 +1,41 @@
+import { createContext, useEffect, useState } from 'react';
+import { apiPath } from '../api'; 
+
+const MyContext = createContext();
+
+const MyContextProvider = ({ children }) => {
```

Comment:
Rename to something more meaningful For instance, using `TeamDataContext` and `TeamDataProvider`